diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -529,18 +549,22 @@ namespace diff_drive_controller
       }
     }
 
+    // Compute/Set control period and frequency (desired/expected or real):
+    const double control_period    = control_period_desired_    > 0.0 ? control_period_desired_    : period.toSec();
```

Comment:
control_period_desired_ seems redundant, why not just do:
`const double control_period    = control_frequency_desired_    > 0.0 ? 1.0 /control_frequency_desired_    : period.toSec();`